src/sass/app.scss

```diff
@@ -1,41 +1,41 @@
-@import "../../node_modules/govuk-frontend/govuk/all.scss";
-@import "variables.scss";
-@import "components/button.scss";
-@import "components/progression.scss";
```

Comment:
I think `npm run lint:fix` is disagreeing with someone's auto format. I'll ad a ticket in case we get time.